From cb1767f233c875654e89b504c6d8d032496fa557 Mon Sep 17 00:00:00 2001
From: Christian Fries <christian.fries123@gmail.com>
Date: Sun, 20 Mar 2016 13:08:13 +0100
Subject: [PATCH] dont seek intl

---
 project/cmake/modules/FindPython.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/project/cmake/modules/FindPython.cmake b/project/cmake/modules/FindPython.cmake
index 6f9f236..93cb12d 100644
--- a/project/cmake/modules/FindPython.cmake
+++ b/project/cmake/modules/FindPython.cmake
@@ -18,7 +18,7 @@ if(NOT PYTHON_FOUND)
 
     find_library(FFI_LIBRARY ffi)
     find_library(EXPAT_LIBRARY expat)
-    find_library(INTL_LIBRARY intl)
+    #find_library(INTL_LIBRARY intl)
 
     if(NOT CORE_SYSTEM_NAME STREQUAL android)
         set(PYTHON_DEP_LIBRARIES -lpthread -ldl -lutil)
-- 
1.9.1

