From 6644369e4a9dffd38d29f50000e311d8554a114a Mon Sep 17 00:00:00 2001
From: Christian Fries <christian.fries123@gmail.com>
Date: Sun, 20 Mar 2016 10:40:05 +0100
Subject: [PATCH] dont do static detection

---
 project/cmake/modules/FindCurl.cmake | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/project/cmake/modules/FindCurl.cmake b/project/cmake/modules/FindCurl.cmake
index a2e8862..e04d73f 100644
--- a/project/cmake/modules/FindCurl.cmake
+++ b/project/cmake/modules/FindCurl.cmake
@@ -22,13 +22,13 @@ if(CURL_FOUND)
     set(CURL_LIBRARY_DIRS ${CURL_LIBDIR})
   endif()
 
-  find_soname(CURL)
+#  find_soname(CURL)
 
-  if(EXISTS "${CURL_LIBRARY_DIRS}/${CURL_SONAME}")
-    execute_process(COMMAND readelf -s ${CURL_LIBRARY_DIRS}/${CURL_SONAME} COMMAND grep CRYPTO_set_locking_call OUTPUT_VARIABLE HAS_CURL_STATIC)
-  else()
-    message(FATAL_ERROR "curl library not found")
-  endif()
+#  if(EXISTS "${CURL_LIBRARY_DIRS}/${CURL_SONAME}")
+#    execute_process(COMMAND readelf -s ${CURL_LIBRARY_DIRS}/${CURL_SONAME} COMMAND grep CRYPTO_set_locking_call OUTPUT_VARIABLE HAS_CURL_STATIC)
+#  else()
+#    message(FATAL_ERROR "curl library not found")
+#  endif()
 endif()
 
 if(HAS_CURL_STATIC)
-- 
1.9.1

