From 4c00219151ffa42f6cfaeaba82813e9099ef5338 Mon Sep 17 00:00:00 2001
From: Christian Fries <christian.fries123@gmail.com>
Date: Sun, 20 Mar 2016 12:26:06 +0100
Subject: [PATCH] smbclient dont use host system

---
 project/cmake/modules/FindLibSmbClient.cmake | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/project/cmake/modules/FindLibSmbClient.cmake b/project/cmake/modules/FindLibSmbClient.cmake
index 897fa7b..d6f0c5c 100644
--- a/project/cmake/modules/FindLibSmbClient.cmake
+++ b/project/cmake/modules/FindLibSmbClient.cmake
@@ -13,20 +13,14 @@ else (LIBSMBCLIENT_LIBRARIES AND LIBSMBCLIENT_INCLUDE_DIRS)
     NAMES
       libsmbclient.h
     PATHS
-      /usr/include
-      /usr/local/include
-      /opt/local/include
-      /sw/include
+      ${LIBSMBCLIENT_INCLUDEDIR}
   )
 
   find_library(SMBCLIENT_LIBRARY
     NAMES
       smbclient
     PATHS
-      /usr/lib
-      /usr/local/lib
-      /opt/local/lib
-      /sw/lib
+      ${LIBSMBCLIENT_LIBDIR}
   )
 
   if (SMBCLIENT_LIBRARY)
-- 
1.9.1

