From 7154bfc85bac199070cbb7a69fdedaeca59ef456 Mon Sep 17 00:00:00 2001
From: Christian Fries <christian.fries123@gmail.com>
Date: Sun, 20 Mar 2016 11:29:06 +0100
Subject: [PATCH] ass_crosscompile

---
 project/cmake/modules/FindASS.cmake | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/project/cmake/modules/FindASS.cmake b/project/cmake/modules/FindASS.cmake
index 5f4b0c7..748e53d 100644
--- a/project/cmake/modules/FindASS.cmake
+++ b/project/cmake/modules/FindASS.cmake
@@ -6,12 +6,18 @@
 # ASS_LIBRARIES - The libass libraries
 
 if(PKG_CONFIG_FOUND)
-  pkg_check_modules (ASS libass)
+  pkg_check_modules (PC_ASS libass)
 else()
   find_path(ASS_INCLUDE_DIRS ass/ass.h)
   find_library(ASS_LIBRARIES NAMES ass libass)
 endif()
 
+find_path(ASS_INCLUDE_DIRS ass.h
+                            PATH_SUFFIXES ass
+                            PATHS ${PC_ASS_INCLUDEDIR})
+find_library(ASS_LIBRARIES NAMES ass libass
+                            PATHS ${PC_ASS_LIBDIR})
+
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(ASS DEFAULT_MSG ASS_INCLUDE_DIRS ASS_LIBRARIES)
 
-- 
1.9.1

