From deb21d01a1c16a125024c2f2c1d71441d26ceda8 Mon Sep 17 00:00:00 2001
From: Christian Fries <christian.fries123@gmail.com>
Date: Sun, 20 Mar 2016 11:06:19 +0100
Subject: [PATCH] use pkginfo

---
 project/cmake/modules/FindCurl.cmake | 11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/project/cmake/modules/FindCurl.cmake b/project/cmake/modules/FindCurl.cmake
index a2e8862..7652e4f 100644
--- a/project/cmake/modules/FindCurl.cmake
+++ b/project/cmake/modules/FindCurl.cmake
@@ -6,12 +6,17 @@
 # CURL_LIBRARIES - The libcurl libraries
 
 if(PKG_CONFIG_FOUND)
-  pkg_check_modules (CURL libcurl)
-  list(APPEND CURL_INCLUDE_DIRS ${CURL_INCLUDEDIR})
+  pkg_check_modules (PC_CURL libcurl)
 else()
-  find_path(CURL_INCLUDE_DIRS curl/curl.h)
   find_library(CURL_LIBRARIES NAMES curl libcurl)
 endif()
+
+find_path(CURL_INCLUDE_DIRS curl.h
+                            PATH_SUFFIXES curl
+                            PATHS ${PC_CURL_INCLUDEDIR})
+find_library(CURL_LIBRARIES NAMES curl libcurl
+                            PATHS ${PC_CURL_LIBDIR})
+
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(Curl DEFAULT_MSG CURL_INCLUDE_DIRS CURL_LIBRARIES)
 
-- 
1.9.1

